pkg/minikube/update/update.go

```diff
@@ -90,11 +95,13 @@ automatically update from %s%s to %s%s now? [y/N] `,
 
 		if strings.ToLower(confirm) == "y" {
 			fmt.Printf("Updating to version %s\n", latestVersion)
-			updateBinary(latestVersion, downloadBinary, updateLinkPrefix, downloadLinkFormat)
+			updateBinary(latestVersion, downloadTarball, updateLinkPrefix, downloadLinkFormat)
 			return
 		}
 
-		fmt.Println("Skipping auto-update.")
+		fmt.Println("Skipping update.")
+	} else {
+		fmt.Println("Already using latest version.")
```

Comment:
I think we should mention the version too. so that user will know which is the latest version here.